packages/devkit/src/mathml.js

```diff
@@ -276,35 +276,31 @@ export default class MathML {
 
   /**
    * Removes semantics tag to mathml.
+   * When using Hand to create formulas, it adds the mrow tag due to the semantics one, this one is also removed.
    * @param {string} mathml - MathML string.
    * @returns {string} - 'mathml' without semantics tag.
    */
   static removeSemantics(mathml) {
-    const mathTagEnd = '</math>';
-    const openSemantics = '<semantics>';
-    const openAnnotation = '<annotation';
-
-    let mathmlWithoutSemantics = mathml;
-    const startSemantics = mathml.indexOf(openSemantics);
-    if (startSemantics !== -1) {
-      const startAnnotation = mathml.indexOf(openAnnotation, startSemantics + openSemantics.length);
-      if (startAnnotation !== -1) {
-        mathmlWithoutSemantics = mathml.substring(0, startSemantics)
-        + mathml.substring(startSemantics + openSemantics.length, startAnnotation) + mathTagEnd;
-      }
-    }
+    // If `mrow` is found right before the `semantics` starting tag, it's removed as well 
+    const semanticsStartingTagRegex = /<semantics>[\n\t]*(<mrow>)?/g;
```

Comment:
Why `[\n\t]` instead of e.g. `\s`?

Reply:
I wasn´t aware `\s` works the same as `\n\t`, in that case it should be replace for the sake of simplicity!